app/src/test/java/org/mozilla/fenix/components/history/PagedHistoryProviderTest.kt

```diff
@@ -57,7 +57,7 @@ class PagedHistoryProviderTest {
             key = historyMetadataKey1,
             title = "mozilla",
             createdAt = 5,
-            updatedAt = 5,
+            updatedAt = 10,
```

Comment:
^^ Just changing the `updatedAt` values because I want the test to fail if we ever no use `createdAt`. :)